witness/golang/omniwitness/usbarmory/main.go

```diff
@@ -64,9 +67,36 @@ func init() {
 	debugConsole, _ := usbarmory.DetectDebugAccessory(250 * time.Millisecond)
 	<-debugConsole
 
+	dcp.Init()
+
 	if err := usbarmory.MMC.Detect(); err != nil {
 		glog.Exitf("Failed to detect MMC: %v", err)
 	}
+
+}
+
+func dcpHasher(r io.Reader) ([32]byte, error) {
+	var ret [32]byte
+	h, err := dcp.New256()
+	if err != nil {
+		return ret, fmt.Errorf("failed to create DCP hasher: %v", err)
+	}
+	bs := 32 << 10
+	buf := make([]byte, bs)
+	var n int
+	for err == nil {
+		n, err = r.Read(buf)
+		if n > 0 {
+			h.Write(buf[:n])
+		}
+
```

Comment:
nit: blank line considered confusing :-)

Reply:
that was to add a delay each time around the loop :)

Reply:
LGTM.